system/kube-system/charts/ingress/templates/ingress-controller-deployment.yaml

```diff
@@ -1,35 +1,47 @@
 apiVersion: extensions/v1beta1
-kind: DaemonSet 
-
+kind: Deployment
 metadata:
   name: ingress-controller
-
+  namespace: kube-system
+  labels:
+    app: ingress-controller
 spec:
```

Comment:
Grr.. Yes. We already talked about this. Better now with bd1bb1d0438e5ca5762c19063e4be40611c737ac ?